### ISSUE TITLE
AI for radiographic COVID-19 detection selects shortcuts over signal

### ISSUE THREAD

**Issue by linahourieh** [linahourieh/machine-learning-articles#12]
## TL;DR
deep learning systems that detect COVID-19 from chest radiographs rely on confounding factors
rather than medical pathology to discriminate healthy versus COVID-19 patients

### Article Link
file:///home/lina/Downloads/s42256-021-00338-7.pdf 

### Author
USA researchers - Nature Article

## Key Takeaways
* The **trustworthiness** of these high performing models needs to be evaluated rigorously before deployment in a clinical setting.
* Researchers proved that **models fail** to learn the true underlying pathology reflecting
the presence of COVID-19 and instead **leverage spurious associations** between the presence or absence of COVID-19 and radiographic features that reflect variations in image acquisition, that is,**shortcuts**.
*  **How??** perspectives of both generalization performance and model explainability. 

* **What they did?**
training deep convolutional neural networks  ---->  Test performance on new hospitals systems --> interrogate the extent to which
using same manners that were                                                                                                           models rely on confounding factors 
mentioned in publications                                                                                                                    saliency maps and generative 
                                                                                                                                                             adversarial networks (GANs)

---> methods to avoid this problem = data quality

* **Build 2 Datasets**: Internal & External Testing: 
Expectation that: a model that relies on valid medical pathology—which should not
change between datasets—should maintain high performance

* a **‘model-centric approach**, which focuses on the specific portions of the radiographs that contribute most to the predictions of our models, in particular, we built **saliency maps** 
*  **data-centric approach**, focusing on the key aspects of the data that could be used to distinguish COVID-19-positive and COVID-19-negative cases. Specifically, we applied **generative adversarial networks**
* **region-swapping’ experiments** in which we swapped out portions of radiographs that our explainable AI approaches identified as important, with the expectation that changes to truly important regions would have a large impact on our classifiers’ outputs.
* h1 : 
Previous publications have suggested that more complex models—that is, those
with higher capacity—may be particularly prone to learning confounds, so we evaluated the generalization performance of simpler models, including a logistic regression and a simple convolutional neural network architecture, but found that the generalization gap did not improve 

* h2:
* wrong data labeling: 
* However, when we modified the label scheme of BIMCV-COVID-19+ such that radiographs are only labelled positive if a radiologist noted evidence of COVID-19, the generalization gap persisted.

saliency maps:
The saliency
maps frequently highlight **laterality markers** that originate during the radiograph acquisition process., which differ in style between the COVID-19-negative and COVID-19-positive datasets.

Our saliency maps also indicate that the image edges, the diaphragm and the cardiac silhouette are important for our models’ predictions of a patient’s COVID-19 status, although these regions are not among those routinely used by radiologists to assess for COVID-1942 and instead probably reflect dataset-level differences in patient positioning and radiographic projection, that is, the anterior–posterior (AP) view versus posterior–anterior (PA).

GANs:
trained GANs to transform
COVID-19-negative radiographs to resemble COVID-19-positive
radiographs and vice versa.
the GAN increases the radiopacity or radiolucency of the lung fields bilaterally to respectively
add or remove evidence of COVID-19, indicating that neural network models are capable of learning genuine markers of COVID-19
However,
the generative networks frequently add or remove laterality markers
and annotations.  The generative networks additionally
alter the radiopacity of image borders

## Useful Code Snippets
```python
#Add your codes here
import numpy as np
import pandas as pd

```

## Useful Tools
* 
*
